src/model/quotes.js

```diff
@@ -909,7 +911,7 @@ class QuotesModel {
         'fspiop-source': ENUM.Http.Headers.FSPIOP.SWITCH.value,
         'fspiop-http-method': ENUM.Http.RestMethods.PUT,
         'fspiop-uri': fspiopUri
-      })
+      }, true)
```

Comment:
Am I missing something, or is this `true` the last argument to `Object.assign`?

Reply:
And if so, where was it meant to go?

Reply:
Its a mistake, I removed it